hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerHA.java

```diff
@@ -171,22 +173,29 @@ public void init() throws Exception {
      */
     conf.set(OZONE_BLOCK_DELETING_SERVICE_INTERVAL, "10s");
     conf.set(OZONE_KEY_DELETING_LIMIT_PER_TASK, "2");
-    additionalConfiguration();
 
     clusterBuilder = MiniOzoneCluster.newOMHABuilder(conf)
         .setClusterId(clusterId)
         .setScmId(scmId)
         .setOMServiceId(omServiceId)
         .setOmId(omId)
         .setNumOfOzoneManagers(numOfOMs);
-    additionalClusterSettings();
 
-    cluster = (MiniOzoneOMHAClusterImpl)clusterBuilder.build();
+    cluster = (MiniOzoneOMHAClusterImpl) clusterBuilder.build();
     cluster.waitForClusterToBeReady();
     objectStore = OzoneClientFactory.getRpcClient(omServiceId, conf)
         .getObjectStore();
   }
 
+  /**
+   * Apply additional configuration between tests.
+   */
+  @Before
+  public void setupTest() {
```

Comment:
@JyotinderSingh As the cluster is already created before executing the `setupTest()` function, there is no impact on adding these configurations. I think, we can remove this method safely because this was added to speed up the cluster setup for each test case and no functional impact. Now, we are very much better by creating cluster only once for all the test cases.

IMHO, we can safely remove #additionalConfiguration() and #additionalClusterSettings() methods from everywhere in the test classes.

Reply:
Understood, will add these changes